### PR TITLE
Add DefinedCrowd and Vestas

### PR DESCRIPTION
**Pull request by 3cpt**
New companies:

- DefinedCrowd
- Vestas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,14 @@ Below is a list of the typical best tech companies to work with in Porto.
 
 ---
 
+### DefinedCrowd
+
+[Official page](https://www.definedcrowd.com/) - [Glassdoor profile](https://www.glassdoor.co.uk/Overview/Working-at-DefinedCrowd-EI_IE1278802.11,23.htm) - [Careers](https://careers.definedcrowd.com/)
+
+**TL;DR for dummies:** - Focused in AI training and data enrich.
```
Comment (filfreire) [inline README.md]: @oandreeeee you mean "enrichment"?

File: README.md [hunk 2]
```diff
@@ -91,6 +99,14 @@ Below is a list of the typical best tech companies to work with in Porto.
 
 ---
 
+### Vestas
```
Comment (filfreire) [inline README.md:102]: @oandreeeee is this one known?

Comment (3cpt) [reply README.md:102]: Yep, a lot of investment in Porto, with a new office in Lionesa, with a bunch new careers opportunities. 

[Engineering Design Centre](https://www.vestas.com/en/career/porto)

File: README.md (hunk 1 above)
Comment (3cpt) [reply README.md]: Yes

**Review by filfreire — APPROVED** [review]

**Pull request merged**